src/index.html

```diff
@@ -4,10 +4,46 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+        class="card"
+        data-qa="card">
+      <img
+          class="card__screen"
+          src="images/imac.jpeg"
+          alt="Screen"
+      >
+      <div class="card__description">
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="reviews__count">Reviews: 5</span>
+        </div>
+        <div class="card__price">
+          <span class="card__price-title">Price:</span>
+          <div class="card__price-price">$2,199</div>
+        </div>
+        <a
+          href="#"
+          class="card__btn"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
+    </div>
```

Comment:
pay attention to your formatting:
```suggestion
    <div class="card" data-qa="card">
      <img
          class="card__screen"
          src="images/imac.jpeg"
          alt="Screen"
      >
      <div class="card__description">
        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
        
        <p class="card__code">Product code: 195434</p>
        
        <div class="reviews">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
          
          <span class="reviews__count">Reviews: 5</span>
        </div>
        
        <div class="card__price">
          <span class="card__price-title">Price:</span>
          <div class="card__price-price">$2,199</div>
        </div>
        
        <a
          href="#"
          class="card__btn"
          data-qa="hover"
        >
          Buy
        </a>
      </div>
    </div>
```
easier to read now, isn't it?

Reply:
Fixed